### PR TITLE
Complete semester endpoint

### PR DESCRIPTION
**Pull request by Lehren**

### CODE REVIEW

File: src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java [hunk 1]
```diff
@@ -45,6 +45,25 @@ public Response getCurriculumSemesters(@PathParam("curriculum_id") final int cur
...
     /**
+     * Get the complete information of one semester in a curriculum.
+     *
+     * @param curriculumId Identifier of the curriculum.
+     * @param semesterId   Identifier of the semester.
+     * @throws Exception In case the querying goes wrong.
+     * @return A JSON Object containing the complete information of this semester.
+     */
+    @GET
+    @Path("/semesters/{semester_id}")
```
Comment (lukeelten) [inline src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java:56]: If you are referring to one semester by providing the semester id or number, please use singular.

**Review by lukeelten — CHANGES_REQUESTED** [review]
Singular! God damit

File: src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java (hunk 1 above)
Comment (Lehren) [reply src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java:56]: This is defined by @sjoerdbr . 

Also the .../modules/module_id endpoint uses plural too.

**Review by lukeelten — APPROVED** [review]

File: src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java (hunk 1 above)
Comment (lukeelten) [reply src/main/java/org/fsg1/fmms/backend/endpoints/CurriculumEndpoint.java:56]: Forget it

**Pull request merged**